03-amazon_s3.Rmd

```diff
@@ -0,0 +1,244 @@
+# Amazon S3
+
+## What is Amazon S3?
+
+[Amazon S3](https://aws.amazon.com/s3/) is a web-based cloud storage platform. It is one of the primary file storage locations on the Analytical Platform, alongside individual users' home directories.
+
+Amazon S3 should be used to store large files and any files that need to be accessed by muliple users.
+
+Data stored in Amazon S3 can be seamlessly integrated with other AWS services such as Amazon Athena and Amazon Glue.
+
+## Working with Amazon S3 buckets
+
+### Types of buckets
+
+Amazon S3 buckets are separated into two categories on the Analytical Platform:
+
+*   warehouse data sources
+*   app data sources
+
+Warehouse data sources are are suitable for storing files in all cases, except where the files need to be accessed by a webapp. In this case, files should be stored in an app data source.
+
+### Create a new bucket
+
+Standard users can currently only create new warehouse data sources in the Analytical Platform control panel. You cannot create new buckets directly in the Amazon S3 console.
+
+To create a new warehouse data source:
+
+1.  Go to the Analytical Platform [control panel](https://cpanel-master.services.alpha.mojanalytics.xyz).
+2.  Select the __Warehouse data__ tab.
+3.  Select __Create a new warehouse data source__.
+4.  Enter a name for the warehouse data source -- this must be prefixed with '-alpha'.
+5.  Select __Create__.
+
+When you create a new warehouse data source, only you will initially have access. As an admin of the data source, you will be able to add and remove other users from the data access group as required. Further information on managing data access groups can be found in Section \@ref(manage-access-to-a-bucket).
+
+### Access levels
+
+Every bucket has three access levels:
+
+*   Read only
+*   Read/write
+*   Admin
+
+### Request access to a bucket
+
+To gain access to a bucket (warehouse data source or app data source), you must be added to the relevant data access group.
+
+If you know an admin of the bucket you require access to, you can ask them to add you to the data access group.
+
+If you do not know any of the admins of the bucket you require access to, you can ask the Analytical Platform team to add you to the data access group on the [#ap_admin_request](https://asdslack.slack.com/messages/CBLAGCQG6/) Slack channel or by [email](mailto:analytical_platform@digital.justice.gov.uk).
+
+When requesting access to a bucket, you should specify the name of the bucket and the level of access you require.
```

Comment:
Somewhere here I think we need to cover the 'need to know' principle - something like: you should only ask to be added if you need access for your work, and this is agreed by the data's admin, as the proxy of the data owner.

Reply:
Added in 26514c9.